homeassistant/components/fritzbox/sensor.py

```diff
@@ -69,14 +103,32 @@ async def async_setup_entry(
 
 
 class FritzBoxBatterySensor(FritzBoxEntity, SensorEntity):
-    """The entity class for FRITZ!SmartHome sensors."""
+    """The entity class for FRITZ!SmartHome battery sensors."""
 
     @property
     def state(self) -> int | None:
         """Return the state of the sensor."""
         return self.device.battery_level  # type: ignore [no-any-return]
 
 
+class FritzBoxPowerSensor(FritzBoxEntity, SensorEntity):
+    """The entity class for FRITZ!SmartHome power consumption sensors."""
+
+    @property
+    def state(self) -> float | None:
+        """Return the state of the sensor."""
+        return self.device.power / 1000  # type: ignore [no-any-return]
+
+
+class FritzBoxEnergySensor(FritzBoxEntity, SensorEntity):
```

Comment:
Make sure you also define `last_reset` for this sensor. https://developers.home-assistant.io/docs/core/entity/sensor